core/src/test/scala/unit/kafka/server/CreateTopicsRequestTest.scala

```diff
@@ -99,29 +101,33 @@ class CreateTopicsRequestTest extends AbstractCreateTopicsRequestTest {
     )
     validateTopicExists("partial-none")
 
-    // Timeout
-    // We don't expect a request to ever complete within 1ms. A timeout of 1 ms allows us to test the purgatory timeout logic.
-    validateErrorCreateTopicsRequests(topicsReq(Seq(
-      topicReq("error-timeout", numPartitions = 10, replicationFactor = 3)), timeout = 1),
-      Map("error-timeout" -> error(Errors.REQUEST_TIMED_OUT)), checkErrorMessage = false)
-    validateErrorCreateTopicsRequests(topicsReq(Seq(
-      topicReq("error-timeout-zero", numPartitions = 10, replicationFactor = 3)), timeout = 0),
-      Map("error-timeout-zero" -> error(Errors.REQUEST_TIMED_OUT)), checkErrorMessage = false)
-    // Negative timeouts are treated the same as 0
-    validateErrorCreateTopicsRequests(topicsReq(Seq(
-      topicReq("error-timeout-negative", numPartitions = 10, replicationFactor = 3)), timeout = -1),
-      Map("error-timeout-negative" -> error(Errors.REQUEST_TIMED_OUT)), checkErrorMessage = false)
-    // The topics should still get created eventually
-    TestUtils.waitForPartitionMetadata(servers, "error-timeout", 0)
-    TestUtils.waitForPartitionMetadata(servers, "error-timeout-zero", 0)
-    TestUtils.waitForPartitionMetadata(servers, "error-timeout-negative", 0)
-    validateTopicExists("error-timeout")
-    validateTopicExists("error-timeout-zero")
-    validateTopicExists("error-timeout-negative")
+    if (!isKRaftTest()) {
```

Comment:
nit: maybe we can pull this into a separate test case since it is specific to zk and it doesn't seem to depend on the any of the logic above.

Reply:
yes, this is a good idea. done.